crm_claim_rma/__openerp__.py

```diff
@@ -66,6 +66,7 @@
  * Joel Grand-Guillaume <joel.grandguillaume@camptocamp.com>
  * Guewen Baconnier <guewen.baconnier@camptocamp.com>
  * Yannick Vaucher <yannick.vaucher@camptocamp.com>
+ * Ondřej Kuzník <ondrej.kuznik@credativ.co.uk>
 
```

Comment:
Please move description in `README.rst` here is the template: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst